engine/api/project/dao_key.go

```diff
@@ -56,3 +56,22 @@ func DeleteProjectKey(db gorp.SqlExecutor, projectID int64, keyName string) erro
 	_, err := db.Exec("DELETE FROM project_key WHERE project_id = $1 AND name = $2", projectID, keyName)
 	return sdk.WrapError(err, "DeleteProjectKey> Cannot delete key %s", keyName)
 }
+
+func loadBuildinKey(db gorp.SqlExecutor, projectID int64) (k sdk.ProjectKey, err error) {
```

Comment:
why named return ?

Reply:
All return statements precise the returned values
Here it's usefull to init the returned values which are used everywhere

Reply:
ok not really fan but ok, however at this line https://github.com/ovh/cds/pull/1543/files/690a39286457e62bcf846a7b73c1706fbe5a4e56#diff-7f65e53ab8b93430efdc845089d28284R62 you make some kind of variable shadowing